app/Helpers/database/user-permission.php

```diff
@@ -27,13 +28,13 @@ function SetAccountPermissionsJSON(
     $retVal = [];
     sanitize_sql_inputs($actingUser, $targetUser);
 
-    $targetUserData = [];
-    if (!getAccountDetails($targetUser, $targetUserData)) {
+    $targetUserModel = User::where('User', $targetUser)->first();
```

Comment:
The `Model` should ve generally omitted imo as it's implicit. Only in cases where it's ambiguous it makes sense to be added - I know we have a lot occurrences of it being in the variable names which could be removed.
Usually it's done where `user` should be called `username`.

Reply:
You're right.

In https://github.com/RetroAchievements/RAWeb/pull/2289/commits/463b9be1b9891f78b8bc08235ba8bd0410efe399, I've made these changes:

* The `$actingUser` string arg has been renamed to `$actingUsername`.
* The `$targetUser` string arg has been renamed to `$targetUsername`.
* The `$targetUserModel` variable has been renamed to `$targetUser`.